libpod/runtime_pod_linux.go

```diff
@@ -220,56 +209,77 @@ func (r *Runtime) removePod(ctx context.Context, p *Pod, removeCtrs, force bool)
 		}
 	}
 
-	// Start removing containers
-	// We can remove containers even if they have dependencies now
-	// As we have guaranteed their dependencies are in the pod
+	// Remove all containers in the pod from the state.
```

Comment:
Do you mean "state" or "start" here? If state, maybe worded differently. Not sure what you are communicating.

Reply:
State - Libpod calls the store of active containers/pods/etc (in the typical case, a BoltDB database) the state (the interface is named `State`, the files are `boltdb_state.go`, etc)